src/client/views/play.html

```diff
@@ -10,25 +10,161 @@
+			onload="(() => { htmlscript.removeOnerror.call(this); })()"></script>
+		<script defer src="/scripts/game/chess/pieces.js" onerror="htmlscript.callback_LoadingError(event)"
+			onload="(() => { htmlscript.removeOnerror.call(this); })()"></script>
+		<script defer src="/scripts/game/chess/movepiece.js" onerror="htmlscript.callback_LoadingError(event)"
+			onload="(() => { htmlscript.removeOnerror.call(this); })()"></script>
+		<script defer src="/scripts/game/chess/selection.js" onerror="htmlscript.callback_LoadingError(event)"
+			onload="(() => { htmlscript.removeOnerror.call(this); })()"></script>
+		<script defer src="/scripts/game/chess/gamefile.js" onerror="htmlscript.callback_LoadingError(event)"
+			onload="(() => { htmlscript.removeOnerror.call(this); })()"></script>
+		<script defer src="/scripts/game/chess/clock.js" onerror="htmlscript.callback_LoadingError(event)"
+			onload="(() => { htmlscript.removeOnerror.call(this); })()"></script>
+		<script defer src="/scripts/game/chess/variantomega.js" onerror="htmlscript.callback_LoadingError(event)"
+			onload="(() => { htmlscript.removeOnerror.call(this); })()"></script>
+		<script defer src="/scripts/game/chess/specialundo.js" onerror="htmlscript.callback_LoadingError(event)"
+			onload="(() => { htmlscript.removeOnerror.call(this); })()"></script>
+		<script defer src="/scripts/game/chess/checkdetection.js" onerror="htmlscript.callback_LoadingError(event)"
+			onload="(() => { htmlscript.removeOnerror.call(this); })()"></script>
+		<script defer src="/scripts/game/chess/movesscript.js" onerror="htmlscript.callback_LoadingError(event)"
+			onload="(() => { htmlscript.removeOnerror.call(this); })()"></script>
+		<script defer src="/scripts/game/chess/specialmove.js" onerror="htmlscript.callback_LoadingError(event)"
+			onload="(() => { htmlscript.removeOnerror.call(this); })()"></script>
+		<script defer src="/scripts/game/chess/copypastegame.js" onerror="htmlscript.callback_LoadingError(event)"
+			onload="(() => { htmlscript.removeOnerror.call(this); })()"></script>
+		<script defer src="/scripts/game/chess/invites.js" onerror="htmlscript.callback_LoadingError(event)"
+			onload="(() => { htmlscript.removeOnerror.call(this); })()"></script>
+        <script defer src="/scripts/game/rendering/statustext.js" onerror="htmlscript.callback_LoadingError(event)"
+                onload="(() => { htmlscript.removeOnerror.call(this); })()"></script>
+		<script defer src="/scripts/validation.js" onerror="htmlscript.callback_LoadingError(event)"
+			onload="(() => { htmlscript.removeOnerror.call(this); })()"></script>
     </head>
+    <!-- The onload event listener to call main.start() is now within the htmlscript! -->
     <body>
         <header>
             <nav>
                 <a href="/">
                     <p>Home</p>
                 </a>
                 <a href="/play">
-                    <p class="currPage">Play</p>
+                    <p>Play</p>
```

Comment:
This line can be reverted. Without it the "Play" link on the header isn't grey to indicate to you that is the page you're on.

Reply:
already did in my second commit